pkg/workflows/internal/v2/sdkimpl/runtime.go

```diff
@@ -0,0 +1,110 @@
+package sdkimpl
+
+import (
+	"errors"
+	"io"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	valuespb "github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
+)
+
+type CallCapabilityFn func(request *pb.CapabilityRequest) ([]byte, error)
```

Comment:
nit: name return types "(id []byte, err error)". It's not obvious that the byte array is an id.